crates/node/src/node.rs

```diff
@@ -104,7 +107,7 @@ pub struct AlphaNetExecutorBuilder;
 
 impl<Node> ExecutorBuilder<Node> for AlphaNetExecutorBuilder
 where
-    Node: FullNodeTypes<Types: NodeTypes<ChainSpec = ChainSpec>>,
+    Node: FullNodeTypes<Types: NodeTypes<ChainSpec = OpChainSpec>>,
```

Comment:
why this change to opchainspec? last i checked it didnt do anything and was just a super shallow wrapper that we cast down to `ChainSpec` anyway

Reply:
there's more stuff wip, because this now allows us to change the OpChainSpec internals, like no dyn Hardforks

Reply:
gotcha